FV3/gfsphysics/GFS_layer/GFS_typedefs.F90

```diff
@@ -5542,6 +5544,8 @@ subroutine diag_phys_zero (Diag, Model, linit, iauwindow_center)
       Diag%dq3dt    = zero
       call Diag%zhao_carr_emulator%zero()
       call Diag%zhao_carr_physics%zero()
+      call Diag%gscond_physics%zero()
+      call Diag%gscond_physics%zero()
```

Comment:
Should one of these be `call Diag%gscond_emulator%zero()`?

Reply:
Good eye! Fixed.